x-pack/plugins/cases/server/connectors/cases/cases_connector.ts

```diff
@@ -43,7 +43,8 @@ interface GroupedAlerts {
 }
 
 type GroupedAlertsWithOracleKey = GroupedAlerts & { oracleKey: string };
-type GroupedAlertsWithCaseId = GroupedAlertsWithOracleKey & { caseId: string };
+type GroupedAlertsWithOracleRecords = GroupedAlertsWithOracleKey & { oracleRecord: OracleRecord };
```

Comment:
To open new cases for the cases that are closed we need the counter that is stored in the oracle record.